packages-backend/loggers/src/create-error-report-logger.ts

```diff
@@ -0,0 +1,89 @@
+import type { Request, Response, NextFunction } from 'express';
+
+import * as Sentry from '@sentry/node';
+import * as env from './env';
+import redactInsecureRequestHeaders from './utils/redact-insecure-request-headers';
+
+const shouldLog = !env.isTest || process.env.DEBUG === 'true';
+
+type LoggerOptions = {
+  sentry?: {
+    DSN: string;
+    role: string;
+    environment: string;
+  };
+  errorMessageBlacklist?: Array<string | RegExp>;
+};
+
+function createErrorReportLogger(options: LoggerOptions = {}) {
+  // Sentry
+  if (options.sentry) {
```

Comment:
Hm. Personally I find it rather specific that we use Sentry. Do we know if others using this package would too or should we allow for more reporting targets?

Reply:
In my defense, the package says "_Opinionated JSON loggers_" 😜